.github/workflows/main.yml

```diff
@@ -54,9 +56,45 @@ jobs:
         env:
           ENABLE_SEARCH: 'true'
 
+      - name: Initialize Deployment
+        uses: bobheadxi/deployments@v0.6.2
+        id: start-deployment
+        with:
+          step: start
+          token: ${{ secrets.GITHUB_TOKEN }}
+          env: ${{ github.ref_name }}
+
+      - name: Deploy to Vercel
+        id: deploy-to-vercel
+        uses: amondnet/vercel-action@v20
+        with:
+          vercel-project-name: commercetools-docs-kit
+          vercel-token: ${{ secrets.VERCEL_TOKEN }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          # vercel-args: '--prod'  # TODO only of on main branch
+          vercel-org-id: ${{ secrets.VERCEL_ORG_ID}}
+          vercel-project-id: ${{ secrets.VERCEL_PROJECT_ID}}
+          alias-domains: | #Optional
+            docs-kit-sha-${{ github.sha }}-commercetools.vercel.app
+            docs-kit-pr-{{PR_NUMBER}}-commercetools.vercel.app
+            docs-kit-branch-{{BRANCH}}-commercetools.vercel.app
```

Comment:
Do we need all of these? 🤔

Reply:
No, this is already changed to only SHA and branch. I needed to experiment with what's possible and what not.   

We wanted to have predictable URLs to be able to generate links or play around with separate projects for the websites so I took the chance here.   SHA and PR number is enough (one per build, one per branch use case but without the restriction in branch name length.